ingest/src/server_mobile.rs

```diff
@@ -222,15 +222,19 @@ where
         request: Request<CellHeartbeatReqV1>,
     ) -> GrpcResult<CellHeartbeatRespV1> {
         let timestamp = Utc::now();
+        let event = request.into_inner();
 
         if timestamp >= self.cbrs_disable_time {
+            let pubkey = PublicKeyBinary::from(event.pub_key);
+            tracing::info!(
+                ?pubkey,
+                "dropping CellHeartbeatReqV1 because cbrs disable time as passed"
```

Comment:
```suggestion
                "dropping CellHeartbeatReqV1 because cbrs disable time has passed"
```

Reply:
this one is confusing b/c it doesn't look like we're actually dropping it, conditionally or otherwise; we're just logging that message regardless of the source of the heartbeat?

Reply:
Might want ton use:
```
custom_tracing::record(
    "pub_key",
    bs58::encode(pubkey).into_string(),
);
```